bears/ruby/RuboCopBear.py

```diff
@@ -28,8 +28,186 @@ def create_arguments(filename, file, config_file, rubocop_config: str=""):
+        :param ignore_unused_method_args_if_empty:
+            Allows unused method argument if method is empty.
+        :param allow_unused_method_keyword_args:
+            Allows unused keyword arguments in a method.
+        """
+
+        options = {
+                  'Style/AccessModifierIndentation':
+                      {'EnforcedStyle': access_modifier_indentation},
+                  'Style/Alias':
+                      {'EnforcedStyle': preferred_alias},
+                  'Style/AlignHash':
+                      {'EnforcedHashRocketStyle': align_hash_rocket_by,
+                       'EnforcedColonStyle': align_colon_by,
+                       'EnforcedLastArgumentHashStyle':
+                           inspect_last_argument_hash},
+                  'Style/AlignParameters':
+                      {'EnforcedStyle': align_parameters},
+                  'Style/ClassCheck':
+                      {'EnforcedStyle': class_check},
+                  'Style/CommentAnnotation':
+                      {'Keywords': comment_keywords},
+                  'Style/GuardClause':
+                      {'MinBodyLength': min_if_unless_lines},
+                  'Style/IndentationWidth':
+                      {'Width': indentation_width},
+                  'Style/MethodName':
+                      {'EnforcedStyle': method_name},
+                  'Style/StringLiterals':
+                      {'EnforcedStyle': string_literals},
+                  'Style/VariableName':
+                      {'EnforcedStyle': variable_name},
+                  'Metrics/ClassLength':
+                      {'Max': max_class_length,
+                       'CountComments': class_length_count_comments},
+                  'Metrics/ModuleLength':
+                      {'CountComments': module_length_count_comments,
+                       'Max': max_module_length},
+                  'Metrics/LineLength':
+                      {'Max': max_line_length,
+                       'AllowHeredoc': line_length_allow_here_doc,
+                       'AllowURI': line_length_allow_uri},
+                  'Metrics/MethodLength':
+                      {'CountComments':  method_count_comments,
+                       'Max': max_method_length},
+                  'Metrics/ParameterLists':
+                      {'Max': max_parameters,
+                       'CountKeywordArgs': count_keyword_args},
+                  'Lint/UnusedBlockArgument':
+                      {'IgnoreEmptyBlocks': ignore_unused_block_args_if_empty,
+                       'AllowUnusedKeywordArguments':
+                           allow_unused_block_keyword_arguments},
+                  'Lint/UnusedMethodArgument':
+                      {'AllowUnusedKeywordArguments':
+                           allow_unused_method_keyword_args,
+                       'IgnoreEmptyMethods':
+                           ignore_unused_method_args_if_empty},
+                  'Metrics/CyclomaticComplexity':
```

Comment:
style... style... metrics... metrics... lint... lint... metrics. huh?

maybe this can also be sorted in with the other metrics?